src/lumigo_tracer/utils.py

```diff
@@ -83,6 +85,7 @@ def _get_event_base64_size(event) -> int:
 def _create_request_body(
     msgs: List[dict], prune_size_flag: bool, max_size: int = MAX_SIZE_FOR_REQUEST
 ) -> str:
+    msgs = omit_keys(msgs)  # extra validation
```

Comment:
Just making sure, this is the omit of the entire span?

Reply:
yep